src/ui/theme.js

```diff
@@ -123,6 +123,8 @@ const theme = {
     },
     textArea: {
       borderColor: colors.grey3,
+      paddingX: '0.84rem',
+      paddingY: '0.44rem',
```

Comment:
So you would not add the padding to the TextArea?
If so I will add that remark to the ticket for @corneelwille

Reply:
Also `paddingX` and `paddingY` can be added directly as props on the component, since Box has support for these props.

So it would look similar to this:

```
paddingX={3}
paddingY={2}
```

Reply:
Behind the scenes value 3 and 2 will get parsed by `parseSpacing` that lives in `Box.js`

Reply:
I used `paddingX` and `paddingY` now and set default values.